pydash/pydash_web/api_routes.py

```diff
@@ -95,3 +95,15 @@ def get_visitor_heatmap(dashboard_id):
 @login_required
 def get_unique_visitor_heatmap(dashboard_id):
     return controller.visitor_heatmap(dashboard_id, 'unique_visitors')
+
+
+@api.route("/api/dashboards/<dashboard_id>/execution_times_per_version", methods=["GET"])
+@login_required
+def get_response_time_per_version_dashboard(dashboard_id):
```

Comment:
Response time?

Reply:
Right, I'll get to rewriting those names in a jiffy. :+1: